### ISSUE TITLE
Rest pain

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10106]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
Rest pain is a continuous unrelenting pain due to ischemia of the lower leg, beginning with or being aggravated by elevation and being relieved by sitting with legs in a dependent position or by standing.

PMID: 28613608 "Patients initially present with the foot, leg, arm, or hand claudication which may be mistaken for joint or neuromuscular problems. Progression of the disease leads to calf claudication, and eventually, ischemic **rest pain** and ulcerations on the toes, feet, or fingers. This is also called Raynaud phenomenon.

PMID: 25770636 "The disease usually begins in the small distal vessels, resulting in distal ischemia (Fig. 1). This explains why most patients present with either **rest pain** involving the hands or feet, digital or toe ulceration, and why up to 40% of patients present with (secondary) Raynaud phenomenon. "

Define critical limb ischemia (CLI): CLI potentially threatens the viability of the limb. Symptoms include **rest pain**, typically occurring at night when the patient is supine and gravity contribution to foot arterial pressure is no longer present. This pain is relieved with foot dependency.
Citation: Arterial Insufficiency 
Lisa S. Foley, Charles J. Fox,
Chapter 73 - Arterial Insufficiency,
Editor(s): Alden H. Harken, Ernest E. Moore,
Abernathy's Surgical Secrets (Seventh Edition),
Elsevier,
2018,
Pages 339-344,
ISBN 9780323478731,
https://doi.org/10.1016/B978-0-323-47873-1.00073-5.
(https://www.sciencedirect.com/science/article/pii/B9780323478731000735)

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Buerger disease (Thromboangiitis obliterans) ORPHA:36258

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000627
- label: Rest limb pain
- parents: [Abnormal arterial physiology (HP:0025323)](https://hpo.jax.org/app/browse/term/HP:0025323)
- pmid: PMID:28613608|PMID:25770636
- def: Rest pain is a continuous unrelenting pain due to ischemia of the lower leg, beginning with or being aggravated by elevation and being relieved by sitting with legs in a dependent position or by standing.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10106
- synonyms: Critical limb ischemia(NONE)
